modules/tests/src/test/scala/ListTest.scala

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2020 by Rob Norris
+// This software is licensed under the MIT License (MIT).
+// For more information see LICENSE or https://opensource.org/licenses/MIT
+
+package tests
+
+import skunk._
+import skunk.codec.all._
+import skunk.implicits._
+
+case object ListTest extends SkunkTest {
+ 
+ // Regression test for https://github.com/tpolecat/skunk/issues/148
+  sessionTest("Do not throw StackOverflowError on big list encoders") { s =>
+    val aLotOfStrings = List.fill(Short.MaxValue)("foo")
```

Comment:
@tpolecat Unrelated: An error is thrown when the list length is bigger than 16 bit integer, e.g. replacing `Short.MaxValue` with `Short.MaxValue + 1` leads to this:
`java.lang.IllegalArgumentException: 32768 is greater than maximum value 32767 for 16-bit signed integer`.

Is there a particular reason for disallowing bigger lists?